### PR TITLE
fix: fix address formatter for starknet address

### PR DESCRIPTION
**Pull request by wa0x6e**
This PR skip the address formatting when it is a starknet address, and also add/fix more tests

### CODE REVIEW

File: src/resolvers/space-sx.ts [hunk 1]
```diff
@@ -15,13 +16,18 @@ const SUBGRAPH_URLS = [
...
+  }
+
   const data = await axios({
     url,
     method: 'POST',
     data: {
       query: `
         query {
-          spaces(where: { id_in: ["${key}", "${getAddress(key)}"] }) {
+          spaces(where: { id_in: ["${id.join('", "')}"] }) {
```
Comment (ChaituVR) [inline src/resolvers/space-sx.ts:30]: maybe this is much better
```suggestion
          spaces(where: { id_in: [${id.map(item => `"${item}"`).join(', ')}] }) {
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Looks like test cases are still failing some how 🤔

**Comment by wa0x6e** [comment]
> Looks like test cases are still failing some how 🤔

Failing tests are unrelated, they're now failing due to something wrong with the starknet library itself, and not due to tests themselves, issue has been open https://github.com/snapshot-labs/stamp/issues/230

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**